### PR TITLE
Усачева Юлия, 2536

### PR DESCRIPTION
**Pull request by littlesun96**
Пришлось попросить сделать push с другого компьютера, потому что на своем не смогла разобраться с проблемой: push git origin master выдает ошибку fatal: Unable to look up github.com (port 9418)

### CODE REVIEW

**Comment by thevery** [comment]
Have you tried both https and ssh protocols?

**Comment by littlesun96** [comment]
Yes, I've tried https, ssh and git. 'git clone' doesn't work too.
git

**Comment by thevery** [comment]
Set GIT_CURL_VERBOSE=1 and try again - this should give some debug info.
Alternatively you can checkout using Android Studio: 
2014-09-14 23-42-59 android studio

**Comment by littlesun96** [comment]
May I come after lessons and ask about this?

**Comment by thevery** [comment]
Sure!

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -1,113 +1,124 @@
...
+                }
+            }
+        }
+        field = field2;
+    }
+
+
+    @Override
+    public void onDraw(Canvas canvas) {
+        Bitmap bitmap = Bitmap.createBitmap(newField, width, height, Bitmap.Config.RGB_565);
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:121]: bitmap re-creation is an expensive operation

**Comment by thevery** [comment]
ok = 8
~20 FPS

**Pull request closed without merging**